frontend/studio-vite/src/core/state/store.ts

```
@@ -0,0 +1,320 @@
+  public getInstrumentManager(): InstrumentManager {
+    return this.instrumentManager;
+  }
+
+  public getMidiManager(): MidiManager {
+    return this.midiManager;
+  }
+
+  /**
+   * Add a listener for track changes
+   */
+  public addListener(listener: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)